Trip planning - NPE while using custom api urls

**Summary:** 

We are getting NPE when using a custom api url.

```

java.lang.RuntimeException: Unable to resume activity {com.joulespersecond.seattlebusbot/org.onebusaway.android.ui.PreferencesActivity}: java.lang.NullPointerException: Attempt to invoke interface method 'java.lang.String org.onebusaway.android.io.elements.ObaRegion.getOtpBaseUrl()' on a null object reference
   at android.app.ActivityThread.performResumeActivity(ActivityThread.java:3103)
   at android.app.ActivityThread.handleResumeActivity(ActivityThread.java:3134)
   at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2481)
   at android.app.ActivityThread.-wrap11(ActivityThread.java)
   at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1344)
   at android.os.Handler.dispatchMessage(Handler.java:102)
   at android.os.Looper.loop(Looper.java:148)
   at android.app.ActivityThread.main(ActivityThread.java:5417)
   at java.lang.reflect.Method.invoke(Native Method)
   at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:726)
   at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:616)
Caused by: java.lang.NullPointerException: Attempt to invoke interface method 'java.lang.String org.onebusaway.android.io.elements.ObaRegion.getOtpBaseUrl()' on a null object reference
   at org.onebusaway.android.ui.PreferencesActivity.onResume(PreferencesActivity.java:165)
   at android.app.Instrumentation.callActivityOnResume(Instrumentation.java:1258)
   at android.app.Activity.performResume(Activity.java:6327)
   at android.app.ActivityThread.performResumeActivity(ActivityThread.java:3092)
   at android.app.ActivityThread.handleResumeActivity(ActivityThread.java:3134) 
   at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2481) 
   at android.app.ActivityThread.-wrap11(ActivityThread.java) 
   at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1344) 
   at android.os.Handler.dispatchMessage(Handler.java:102) 
   at android.os.Looper.loop(Looper.java:148) 
   at android.app.ActivityThread.main(ActivityThread.java:5417) 
   at java.lang.reflect.Method.invoke(Native Method) 
   at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:726) 
   at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:616) 

```

**Steps to reproduce:** 

1. Go to `Settings`
1. Set custom api url
1. Go to main screen
1. Tap on `Settings` again

**Expected behavior:** 

No NPE

**Observed behavior:** 

NPE

**Device and Android version:** 

Motorola Nexus 6 w/ Android 6.0.1

**Screenshots:** 
N/A

--- a/onebusaway-android/src/main/java/org/onebusaway/android/ui/PreferencesActivity.java
+++ b/onebusaway-android/src/main/java/org/onebusaway/android/ui/PreferencesActivity.java
@@ -161,8 +161,8 @@ public class PreferencesActivity extends PreferenceActivity
         changePreferenceSummary(getString(R.string.preference_key_preferred_units));
 
         // Remove preferences for notifications if no trip planning
-        String otpBaseUrl = Application.get().getCurrentRegion().getOtpBaseUrl();
-        if (TextUtils.isEmpty(otpBaseUrl)) {
+        ObaRegion obaRegion = Application.get().getCurrentRegion();
+        if (obaRegion!= null && TextUtils.isEmpty(obaRegion.getOtpBaseUrl())) {
             PreferenceCategory notifications = (PreferenceCategory)
                     findPreference(getString(R.string.preference_key_notifications));
 

